web/templates/settings/organization_permissions_admin.hbs

```diff
@@ -353,6 +353,10 @@
                       label=group_setting_labels.can_summarize_topics_group}}
                 </div>
 
+                {{> group_setting_value_pill_input
+                  setting_name="realm_can_manage_billing_group"
+                  label=group_setting_labels.can_manage_billing_group}}
```

Comment:
I think this should go above the AI setting, looking at the the issue specification.